docs/source/inference.mdx

```diff
@@ -99,21 +99,22 @@ tokenizer.save_pretrained(save_directory)
 
 ### Weight-only quantization
 
-You can also apply 8-bit or 4-bit weight quantization when exporting your model with the CLI by setting the `weight-format` argument to respectively `int8` or `int4`:
+You can also apply fp16, 8-bit or 4-bit weight quantization on the linear and embedding layers when exporting your model with the CLI by setting `--weight-format` to respectively `fp16`, `int8` or `int4`:
```

Comment:
convolutional layers are supported as well

Reply:
added it in https://github.com/huggingface/optimum-intel/pull/601/commits/be866d49e75cd83771d0bc990760d05ef48e7b92